client/state/jetpack-review-prompt/selectors.ts

```diff
@@ -14,12 +15,14 @@ const getExistingPreference = (
 	type: 'scan' | 'restore'
 ): SinglePreferenceType => {
 	const pref = ( getPreference( state, PREFERENCE_NAME ) as PreferenceType ) || {};
-	if ( type === 'scan' ) {
-		return pref.hasOwnProperty( 'scan' )
-			? pref.scan[ state.ui.selectedSiteId ] ?? emptyPreference
-			: emptyPreference;
+	if ( type === 'scan' && pref?.scan ) {
+		return (
```

Comment:
Minor, but since you added the `pref?.scan` guard above, this return statement could be rewritten as:

```
		const scanKey = state.ui.selectedSiteId as keyof ScanPreferenceType;
		return pref.scan[ scanKey ] ?? emptyPreference;
```

Reply:
Ahh, nice!  Thank you! 👍